model/fevm/contract.go

```diff
@@ -0,0 +1,46 @@
+package fevm
+
+import (
+	"context"
+
+	"go.opencensus.io/tag"
+
+	"github.com/filecoin-project/lily/metrics"
+	"github.com/filecoin-project/lily/model"
+)
+
+type FEVMContract struct {
+	tableName struct{} `pg:"fevm_contract"` // nolint: structcheck
```

Comment:
```suggestion
	tableName struct{} `pg:"fevm_contracts"` // nolint: structcheck
```

We should make this consistent as with other models in Lily. They're all plural and it makes sense as in a table you don't have just a single contract but multiple.

Reply:
changed